src/riak_kv_keys_fsm.erl

```diff
@@ -94,8 +102,26 @@ init(From={_, _, ClientPid}, [Bucket, ItemFilter, Timeout]) ->
     %% Construct the key listing request
     Req = req(Bucket, ItemFilter),
     {Req, all, NVal, 1, riak_kv, riak_kv_vnode_master, Timeout,
+     #state{from=From}};
+
+init(From={_, _, ClientPid}, [Table, ItemFilter, Timeout, DDL]) ->
```

Comment:
Can this duplication of the `init` function be avoided at the riak_client level?  For example, passing an `no_ddl` atom for the DDL instead of no element for it.